packages/manager/src/views/WebPropertyDetailPage/components/SSR/ViewLogs.tsx

```diff
@@ -0,0 +1,137 @@
+import { fetchLogsforSpa, useListOfPods } from '@app/services/appLogs';
+import {
+  CodeBlock,
+  EmptyState,
+  EmptyStateIcon,
+  Select,
+  SelectOption,
+  Title
+} from '@patternfly/react-core';
+import { CubesIcon } from '@patternfly/react-icons';
+import { useEffect, useState } from 'react';
+
+type Props = {
+  spaName: string;
+  propertyIdentifier: string;
+  env: string;
+  type: string | number;
+  idList: string[];
+};
+
+export const ViewLogs = ({ propertyIdentifier, spaName, env, type, idList }: Props) => {
+  const [logsData, setLogsData] = useState<any>([]);
+  const [isLogsLoading, setIsLogsLoading] = useState<boolean>(true);
+  const [isOpen, setIsOpen] = useState(false);
+  const [Id, setId] = useState('');
+
+  const podList = useListOfPods(propertyIdentifier, spaName, env);
+
+  const onToggle = (isOpen1: boolean) => {
+    setIsOpen(isOpen1);
+  };
+  const clearSelection = () => {
+    setId('');
+    setIsOpen(false);
+  };
+  const onSelect = (
+    event: React.MouseEvent | React.ChangeEvent,
+    selection: string,
+    isPlaceholder?: boolean
+  ) => {
+    if (isPlaceholder) clearSelection();
+    else {
+      setId(selection);
+    }
+    setIsOpen(false);
+  };
+
+  useEffect(() => {
+    const pID = Id !== '' ? Id : podList?.data && podList?.data[0];
+    const buildID = Id !== '' ? Id : idList && setId(idList.reverse()[0]);
+
+    fetchLogsforSpa(
+      propertyIdentifier,
+      spaName,
+      env,
+      type === 0 ? 'POD' : 'BUILD',
```

Comment:
`const logType = { POD : 'POD', BUILD : 'BUILD }`
and use this throughout the code, repeating in multiple places
and use `logType.POD` or `logType.BUILD`